roles/libvirt_manager/molecule/deploy_layout/converge.yml

```diff
@@ -47,7 +47,7 @@
           extra_disks_num: 1
           extra_disks_size: 1G
         compute:
-          amount: 2
+          amount: 1
```

Comment:
is this related to the 'skip tasks' or 'host specific inventory vars' or something else?

Reply:
no, just that we don't need to create so many nodes "for nothing". Also improves a bit the speed of molecule itself.